tests/test_card.py

```diff
@@ -32,26 +42,45 @@ def test_all(self):
             }
         }
 
-        customer_id = "685810954-vbrXmBzkHl4UJ9"
         card_token_created = self.sdk.card_token().create(card_token_object)
 
         card_object = {
-            "customer_id": customer_id,
+            "customer_id": self._customer_id,
             "token": card_token_created["response"]["id"]
         }
 
-        card_created = self.sdk.card().create(customer_id, card_object)
+        card_created = self.sdk.card().create(self._customer_id, card_object)
         self.assertIn(card_created["status"], [200, 201])
-        self.assertEqual(self.sdk.card()
-        .get(customer_id, card_created["response"]["id"])["status"], 200)
+        self.assertEqual(self.sdk.card().get(self._customer_id, card_created["response"]["id"])["status"], 200)
+
+        self.sdk.card().delete(self._customer_id, card_created["response"]["id"])
+
+    def createCustomer(self):
+        customer_object = {
+            "email": "test_payer_999942@testuser.com",
+            "first_name": "Rafa",
+            "last_name": "Williner",
+            "phone": {
+                "area_code": "03492",
+                "number": "432334"
+            },
+            "identification": {
+                "type": "DNI",
+                "number": "29804555"
+            },
+            "address": {
+                "zip_code": "47807078",
+                "street_name": "some street",
+                "street_number": 123
+            },
+            "description": "customer description"
+        }
+
+        return self.sdk.customer().create(customer_object)
+
+    def deleteCustomer(self):
```

Comment:
Same here

This can be a `@classmethod` since you're accessing a class attribute, by converting it, you don't need to provide a `cls()` as parameter in `tearDownClass` as you're doing.

Reply:
@rmanganiello could review again? Tks